.idea/uiDesigner.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this a necessary file? I don't see it referenced anywhere and looks like it's a Java file